src/Components/ArticleCard.jsx

```diff
@@ -0,0 +1,34 @@
+import CardActions from '@mui/material/CardActions';
+import CardContent from '@mui/material/CardContent';
+import CardMedia from '@mui/material/CardMedia';
+import Button from '@mui/material/Button';
+
+function ArticleCard({articleData}){
+    const {author, title, topic, created_at,votes,article_img_url,comment_count}=articleData
+    const date=created_at.split("T")[0].split("-")
+    const months=["January", "February", "March", "April", "May", "June", 
+           "July", "August", "September", "October", "November", "December" ]
```

Comment:
I would maybe pack this logic into a utils.js file and import it, just so this component isn't clogged up 👍